exec/java-exec/src/main/codegen/data/Parser.tdd

```diff
@@ -36,7 +36,34 @@
     "USE",
     "FILES",
     "REFRESH",
-    "METADATA"
+    "METADATA",
+    "ANALYZE",
+    "COMPUTE",
+    "ESTIMATE",
+    "STATISTICS",
+    "COLUMNS",
+    "SAMPLE"
+    "PERCENT"
+  ]
+
+  # List of keywords from "keywords" section that are not reserved by SQL:2003 standard.
+  # Example: "DATABASES", "TABLES" are keywords but are not reserved by SQL:2003 standard.
+  # First keyword that starts the statement should be a reserved keyword, otherwise the current parser
+  # ends up considering it as a expression and fails (TODO: need to follow up with Calcite to find if it is a bug
+  # or by design.
+  nonReservedKeywords: [
+    "DATABASES",
+    "REPLACE",
+    "SCHEMAS",
+    "TABLES",
+    "FILES",
```

Comment:
Something like this came up before where a list of non reserved keyword might result in some ambiguous queries. See [DRILL-2116](https://issues.apache.org/jira/browse/DRILL-2116).